src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -460,15 +531,28 @@ export class MgtPersonCard extends MgtTemplatedComponent {
         this.personDetails = (parent as MgtPerson).personDetails;
         this.personImage = (parent as MgtPerson).personImage;
       }
-    }
+    } else {
+      // Use userId or 'me' query to get the person and image
+      if (this.userId || this.personQuery === 'me') {
```

Comment:
there is no shared logic between these two if statements, they can be moved one level up as part of the `else statement.